src/components/kepler-gl.js

```diff
@@ -358,14 +353,14 @@ function KeplerGlFactory(
                 <div className="maps" style={{display: 'flex'}}>
                   {mapContainers}
                 </div>
-                {isExporting && (
+                {isExportingImage && (
                   <PlotContainer
                     width={width}
                     height={height}
                     exportImageSetting={uiState.exportImage}
                     mapFields={mapFields}
```

Comment:
i realized the `splitMaps` are not exported as 2 maps. digging into it, you just need to pass in `splitMaps` here into the `PlotContainer` then you will be able to export a splitted image:

<img width="964" alt="Screen Shot 2020-07-23 at 11 39 29 PM" src="https://user-images.githubusercontent.com/3605556/88366346-c29d2880-cd3d-11ea-9893-fd95cf8ac69e.png">